loss-detection-app/pipelines/object_detection/5/pipeline.json

```diff
@@ -1,14 +1,12 @@
 {
-    "name": "object_detection",
-    "version": 1,
     "type": "GStreamer",
-    "template":"v4l2src device=\"{source[path]}\" ! concat name=c ! decodebin ! video/x-raw ! videoconvert name=\"videoconvert\" ! gvadetect inference-id=inf0 model=\"{models[object_detection][1][network]}\" model-proc=\"{models[object_detection][1][proc]}\" name=\"detection\" ! gvaidentify tracker=true ! gvametaconvert converter=json method=detection source=\"{source[path]}\" name=\"jsonmetaconvert\" include-no-detections=true ! gvametapublish method=file filepath=\"/results/results.txt\" ! queue ! appsink name=appsink",
+    "template":"v4l2src device=\"{source[path]}\" ! concat name=c ! decodebin ! video/x-raw ! videoconvert name=\"videoconvert\" ! gvadetect model=\"{models[object_detection][1][network]}\" model-proc=\"{models[object_detection][1][proc]}\" name=\"detection\" ! gvaidentify ! gvametaconvert name=\"metaconvert\" source=\"{source[path]}\" add-empty-results=true ! gvametapublish name=\"destination\" file-path=\"/results/results.txt\" ! queue ! appsink name=appsink",
```

Comment:
Not sure why this was is needed - can we remove it. We support file path in destination via the request.

Reply:
Removed, will document